integreat_cms/cms/templates/pois/poi_form.html

```diff
@@ -156,9 +156,14 @@ <h3 class="heading font-bold text-black">
                         </div>
                         <div>
                             <label>{% trans 'Position' %}</label>
-                            <div class="help-text">
-                                {% trans 'If you leave these fields blank, they are automatically derived from the address.' %}
+                            <div>
+                                <input type="checkbox" id="auto-fill-coordinates" checked>
```

Comment:
```suggestion
                                <input type="checkbox" id="auto-fill-coordinates" data-url="{% url 'get_coordinates' %}" checked>
```
Ah, one additional thing: I would not hardcode the url, but get it via the template tag instead.